client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html

```diff
@@ -1,207 +1,201 @@
-                                        <i class="oui-icon oui-icon-checkbox-indeterminate" aria-hidden="true"></i>
-                                    </span>
-                                </span>
-                            </label>
-                        </div>
-                        
+            <!-- No Service -->
+            <div data-ng-if="!ctrl.availableServices">
+                <p data-i18n-static="email_tab_modal_migrate_no_email_service"></p>
+                <p>
+                    <a class="btn btn-primary" href="{{::ctrl.emailsOrder}}" target="_blank"
+                        title="{{tr('email_tab_modal_migrate_order_email_service')}} ({{tr('core_new_window')}})">
+                        <span data-i18n-static="email_tab_modal_migrate_order_email_service"></span>
+                        <span class="fa fa-external-link"></span>
+                    </a>
+                </p>
+            </div>
+            <!-- /No Service -->
+
+            <!-- Has Services -->
+            <div class="oui-field"
+                data-ng-if="ctrl.availableServices">
+                <label class="oui-field__label oui-label"
+                    data-i18n-static="email_tab_modal_migrate_destination_service_label">
+                </label>
+                <div class="row oui-field__control oui-field__control_auto">
+                    <div class="col-md-{{(12 / ctrl.availableServices)}}"
+                        data-ng-repeat="(serviceType, destinationService) in ctrl.destinationServices track by $index"
+                        data-ng-hide="!ctrl.destinationFocused && (ctrl.migrate.destinationService.type !== serviceType)">
+                        <oui-select-picker name="serviceType"
+                            data-text="{{::serviceType}}"
+                            data-placeholder="{{::tr('select_placeholder')}}"
+                            data-model="ctrl.migrate.destinationService"
+                            data-match="name"
+                            data-values="destinationService">
+                        </oui-select-picker>
                     </div>
                 </div>
+            </div>
+            <!-- /Has Services -->
+
+        </oui-step-form>
+        <!-- /Step 1: Destination -->
+
+        <!-- Step 2: Informations -->
+        <oui-step-form data-header="{{::tr('email_tab_modal_migrate_informations_step_title')}}"
+            data-loading="ctrl.loaders.isWaitingForDestinationEmails"
+            data-valid="ctrl.migrate.destinationEmail"
+            data-navigation="ctrl.migrate.destinationEmail"
+            data-on-focus="ctrl.destinationFocused = false"
+            data-on-submit="ctrl.checkMigrationData()">
+
+            <!-- Has available email address -->
+            <div data-ng-if="ctrl.migrate.destinationEmail">
+
+                <!-- Before -->
+                <h4 data-i18n-static="email_tab_modal_migrate_informations_before_title"></h4>
```

Comment:
Is the `h4`justified ? (Do we have a `h3`, `h2` and `h1` before ?

Reply:
I change it with a `<p  class="oui-header_4">`